### PR TITLE
add: Features section to README

### PR DESCRIPTION
**Pull request by YusukeHosonuma**
Because so cool :)

And add table of contents.

### CODE REVIEW

**Review by po-miyasaka — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -13,6 +13,44 @@ SwiftPrettyPrint gives **Human-readable outputs** than `print()`, `debugPrint()`
 
 ![Screenshot](https://raw.githubusercontent.com/YusukeHosonuma/SwiftPrettyPrint/main/Image/screenshot.png)
 
+## Features 🚀
+
+ - ✅ Style
```
Comment (po-miyasaka) [inline README.md]: 👀
```suggestion
 - Style
```

Comment (YusukeHosonuma) [reply README.md]: Thx!

File: README.md [hunk 2]
```diff
@@ -13,6 +13,44 @@ SwiftPrettyPrint gives **Human-readable outputs** than `print()`, `debugPrint()`
...
+ - Package Manager
+   - ✅ Swift Package Manager
+   - ✅ CocoaPods
+   - ✅ Carthage
+ - OS Support
+   - ✅ Apple platforms
+   - ✅ Linux
+   - [ ] Windows
+
+## Table of Contents 📖
```
Comment (po-miyasaka) [inline README.md:34]: 👍

**Review by po-miyasaka — APPROVED** [review]
LGTM!

**Pull request merged**